app/assets/components/service_update/view.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class ServiceUpdate::View < GovukComponent::Base
+  attr_reader :service_update
+
+  delegate :title, :content, :date, to: :service_update
+
+  TITLE_TAGS = %w[h1 h2 h3 h4 h5 h6].freeze
+  TITLE_CLASS = "govuk-heading-m govuk-!-margin-bottom-2"
+
+  def initialize(service_update:, title_tag: "h2")
+    @service_update = service_update
+    @title_tag = title_tag
+  end
+
+  def render?
+    service_update
+  end
+
+  def title_element
+    if TITLE_TAGS.include?(title_tag)
+      return tag.public_send(title_tag, title, class: TITLE_CLASS)
+    end
+
+    tag.h2(title, class: TITLE_CLASS)
+  end
+
+  def date_pretty
+    date.to_date.strftime("%e %B %Y")
+  end
+
+  def content_html
+    custom_render =
+      Redcarpet::Render::HTML.new(link_attributes: { class: "govuk-link" })
+    Redcarpet::Markdown.new(custom_render).render(content).html_safe
```

Comment:
I think a lot of the heavy lifting can be done using this DfE gem:
https://github.com/DFE-Digital/govuk-markdown

It uses Redcarpet configured to produce HTML with the Design System class names.

For example:

```ruby
GovukMarkdown.render("[A link](/foo)")
```

Will output:

```html
<p class="govuk-body-m">
  <a href="/foo" class="govuk-link">A link</a>
</p>
```

It seems like we could avoid a lot of the manual HTML tag config done within this ViewComponent if we used that instead.